### DISCUSSION TITLE
Need Guidance on How to Contribute

### DISCUSSION THREAD

**Discussion by WajahatKanju** [mirumee/saleor-dashboard#4078] [Q&A]
I'm new to open source and would love to know more about how to get started. What are the steps involved in making a contribution? Are there any specific projects that are welcoming to newcomers? I'm open to coding, documentation, and other types of contributions, so any recommendations or pointers would be highly valuable.

**Comment by tgrycuk** [comment]
Hi, my name is Tom and I'm new here too.
I'm also looking for some instructions on how to properly prepare and execute a pull request.
I was looking for instructions or tips but I couldn't find anything specific.
If there is no such thing, it may be worth preparing it together, e.g. in the CONTRIBUTING.md file.

Regards
